src/test/java/edu/unc/lib/boxc/migration/cdm/services/ArchivalDestinationsServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+import edu.unc.lib.boxc.migration.cdm.options.DestinationMappingOptions;
+import edu.unc.lib.boxc.migration.cdm.test.BxcEnvironmentHelper;
+import edu.unc.lib.boxc.migration.cdm.test.CdmEnvironmentHelper;
+import edu.unc.lib.boxc.migration.cdm.test.SipServiceHelper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+
+public class ArchivalDestinationsServiceTest {
+    private static final String PROJECT_NAME = "proj";
+
+    @TempDir
+    public Path tmpFolder;
+
+    private SipServiceHelper testHelper;
+    private MigrationProject project;
+    private ArchivalDestinationsService service;
+    private IndexFilteringService indexService;
+
+    @BeforeEach
+    public void setup() throws Exception {
+        project = MigrationProjectFactory.createMigrationProject(
+                tmpFolder, PROJECT_NAME, null, "user",
+                CdmEnvironmentHelper.DEFAULT_ENV_ID, BxcEnvironmentHelper.DEFAULT_ENV_ID);
+        testHelper = new SipServiceHelper(project, tmpFolder);
+        service = new ArchivalDestinationsService();
+        indexService = new IndexFilteringService();
```

Comment:
Looks like this isn't being used since it passes in `testHelper.getIndexService()` below

Reply:
There is a `NullPointerException` when I remove those lines and the test fails.